setup.py

```diff
@@ -30,7 +30,7 @@
     },
 
     install_requires=[
-        'pylint==1.4.4',
+        'pylint>=1.5.0,<1.6.0',
```

Comment:
Currently, edx-platform only gets pylint installed because of this dependency.  Developers have complained in the past of the unpredictability of the differences between running pylint locally and on CI machines.  I think it would be best to keep this precisely pinned to reduce variation.